### DISCUSSION TITLE
How does a retraction event work?

### DISCUSSION THREAD

**Discussion by deepisidhu** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#220] [Q&A] [answered]
Hi, I'm trying to understand how to interpret _Retraction_ events. I assume that a retraction event is similar to a cancellation, in that it's used to undo a previous event. If that is correct, how do you reference back to the original event that this retraction is correcting?

>  {
      "object_type": "TX_PLAN_SECURITY_RETRACTION",
      "id": "test-plan-security-retraction-full-fields",
      "security_id": "0f96b82a-6dc5-4205-bcb1-15740e5f8304",
      "date": "2022-01-24",
      "reason_text": "We wish to make a retraction",
      "comments": ["A", "Series", "Of", "Comments"]
    },

**Comment by JSv4** [comment, marked as answer]
Hey @deepisidhu, thanks for the question. Retraction is similar to a cancellation, but was envisioned to work in situations where a security was not actually _accepted_ / _delivered_ for whatever reason. The example some of the vendor teams floated was where you had an option approved and issued to someone but they left before accepting the option.

**Reply by deepisidhu** [reply]
Got it, thanks for the explanation!
